arangod/RocksDBEngine/RocksDBCuckooIndexEstimator.h

```diff
@@ -153,7 +153,8 @@ class RocksDBCuckooIndexEstimator {
 
  public:
   static bool isFormatSupported(arangodb::velocypack::StringRef serialized) {
-    switch (serialized.front()) {
+    TRI_ASSERT(serialized.size() > sizeof(_appliedSeq) + sizeof(char));
+    switch (serialized[sizeof(_appliedSeq)]) {
```

Comment:
mmmh. Previously we switched upon the value of a single character at the first byte of the StringRef. Now we are switching upon the value of a different byte... Is that downwards-compatible, and what is the motivation for the change?

Reply:
I did not design the format, my intention was to put the deserialization code into the cuckoo estimator

Reply:
This should be fine as-is. Just moving the check from one place to another, format hasn't changed.